sdk/communication/Azure.Communication.PhoneNumbers/CHANGELOG.md

```diff
@@ -10,8 +10,9 @@
   - GetPhoneNumbers -> GetPurchasedPhoneNumbers.
   - GetPhoneNumbersAsync -> GetPurchasedPhoneNumbersAsync.
 - PhoneNumbersModelFactory static method AcquiredPhoneNumber is renamed to PurchasedPhoneNumber.
-- PurchasePhoneNumbersOperation extends Operation instead of Operation<Response>.
-- ReleasePhoneNumberOperation extends Operation instead of Operation<Response>.
+- PurchasePhoneNumbersOperation and ReleasePhoneNumberOperation extend Operation instead of Operation<Response>.
+- Added protected constructor to PurchasePhoneNumbersOperation and ReleasePhoneNumberOperation for mocking.
```

Comment:
Should that be added to 'Added' section instead of braking changes?

Reply:
Updated it